<launch>

  <arg name="config_path"
    default="$(find flir_camera_driver)/config/30fps.yaml"/>



  <node pkg="flir_camera_driver" type="flir_camera_driver" name="flir_camera_driver" output="screen">
    <param name="config_file" type="string" value="$(arg config_path)" />
  </node>

</launch>
